opensrp-app/src/main/java/org/smartregister/view/activity/BaseLoginActivity.java

```diff
@@ -231,7 +234,7 @@ public void onClick(View v) {
     }
 
     protected void attemptLogin() {
-        String username = userNameEditText.getText().toString().trim();
+        String username = userNameEditText.getText().toString().trim().toLowerCase(Locale.ENGLISH);
```

Comment:
Isn't the username actually case sensitive?

Reply:
Currently we don't support them as such @dubdabasoduba